--- origsrc/osmo-0.1.2/Makefile.am	2007-11-17 09:27:36.000000000 -0600
+++ src/osmo-0.1.2/Makefile.am	2007-11-17 22:44:11.671875000 -0600
@@ -1,5 +1,4 @@
 SUBDIRS = src po
-ACLOCAL_AMFLAGS = -I m4
 
 pixmapdir = $(datadir)/pixmaps
 pixmap_DATA = osmo.svg osmo.png
--- origsrc/osmo-0.1.2/po/Makefile.am	2007-11-17 09:27:36.000000000 -0600
+++ src/osmo-0.1.2/po/Makefile.am	2007-11-17 22:59:03.250000000 -0600
@@ -15,7 +15,7 @@
 all: $(DOMAIN).pot $(POFILES) $(MOFILES)
 
 %.mo: skip
-	@po=$(@:.mo=.po); if test $$po -nt $@ ; then $(MSGFMT) -c --statistics $$po -o $@; echo "$@ updated."; fi
+	@po=$(@:.mo=.po); if test $(srcdir)/$$po -nt $@ ; then $(MSGFMT) -c --statistics $(srcdir)/$$po -o $@; echo "$@ updated."; fi
 
 %.po: $(DOMAIN).pot
 	$(MSGMERGE) $@ $< -o $@.in && mv $@.in $@
